src/nvim/drawline.c

```diff
@@ -944,6 +944,10 @@ int win_line(win_T *wp, linenr_T lnum, int startrow, int endrow, int col_rows, s
   static char *at_end_str = "";       // used for p_extra when displaying curwin->w_p_lcs_chars.eol
                                       // at end-of-line
   const bool has_fold = foldinfo.fi_level != 0 && foldinfo.fi_lines > 0;
+  const bool has_foldtext = has_fold && *wp->w_p_fdt != NUL;
+
+  const bool wrap = wp->w_p_wrap
```

Comment:
It may be better to move this to `winlinevars_T` so that the `wrap1` below can still be called `wrap`.

Reply:
I'm not sure that really improves anything since you're still left with two variables named `wrap`, and it seems variables are arbitrarily added to `winlinevars_T`.

Is there a better name we can give to `wrap1`? I named it that to give some indication that it is different to `wrap`.

Reply:
I just don't want change the meaning of `wrap` in its original scope to a different variable, as that may lead to mistakes. Maybe name this `is_wrapped` instead?

Reply:
And... I think it's better to leave shorter or more general names for smaller scopes.

Reply:
Done